vernac/declare.mli

```diff
@@ -437,6 +437,16 @@ val declare_constant
   -> constant_entry
   -> Constant.t
 
+(** Like [declare_definition] but also returns the final UState.t *)
```

Comment:
If we do the `if poly` here instead of in the caller we can document as "also returns the universes and constraints added to the global env"

Reply:
ok